### ISSUE TITLE
I confirm this library is retarded

### ISSUE THREAD

**Issue by Porkechebure** [micjahn/ZXing.Net#209]
For anyone coming here by change, save your ass from hair pulling with this shit library

It's retarded to say the least and author is a fag who cries and closes threads when you confront him.

It has a complicated and unpleasant way to generate the barcodes, you will find a ton of versions and end up doing some horrible spaghetti code just to get some image.

Please find something else. I'm not paid by anyone so I won't do names.... but find something else, there are plenty of better options.

https://forums.xamarin.com/discussion/155430/zxing-barcodewriter-issue-string-to-barcode
(NOT MY THREAD)
This is what I had to copy around to just get an image and it's ALL BLACK no barcode in it....
And I had to copy this because getting the latest nuget package you find has a fucking  BarcodeWriter<TOuput> for which he conveniently doesn't give a fucking example to make it work and produce a simple barcode, while all the examples on google bring up a simple

new  BarcodeWriter() without any <TOutput>.

Nice way to introduce changes which breaks every past compatibility, or if it's not that, you did a shit job anyway since it's not even understandable unless I want to dig my head for hours (for which people is paid you know?) to read your horrible spaghetti code and understand what it does.

People in real world need to produce working code fast, not reverse engineer some weird shit.

This guy has no idea of what business world is about and will make you spend 3 days figuring out how this crap works.

Ah, just to show all of you I'm not an angry fool, compare this code from **another** library available around which I won't name:

`    BarcodeSettings setting = new BarcodeSettings();

            setting.Type = BarCodeType.Code128;

            setting.Code128SetMode = Code128SetMode.Auto;

            setting.Data = "ABC123"

            BarCodeGenerator generator = new BarCodeGenerator(setting);

            Image image = generator.GenerateImage();

            image.Save(path + "Code128.png");`

and compare it to the given example up there....

Make your compares, check how simple and straightforward is compared to that bunch of shit in the link above, which, by the way, it's not even the full code needed to produce a working image, you still have to write more code, that is just the "rendering engine" which YOU HAVE TO WRITE ON YOUR OWN

AHAHAHAHHAHAHAHAHAHAHAHAH FEELS LIKE LINUX WHERE YOU HAVE TO COMPILE DRIVERS

Enough said, he will close this too.

Sincerely,
FUCK YOU

**Comment by micjahn** [comment]
It's a free library. Nobody forces you to use it. Feel free to go ahead and use another library (like Spire.Barcode from which your code snippet is from).

**Comment by Porkechebure** [comment]
> It's a free library. Nobody forces you to use it. Feel free to go ahead and use another library (like Spire.Barcode from which your code snippet is from).

Instead of sending people away give a use to your library and provide working examples

**Comment by micjahn** [comment]
Definitely not if the issue is opened with the words "shit library" and the last words are "FUCK YOU". If you ask a question in a respectful manner you will receive a constructive answer.

**Comment by Porkechebure** [comment]
> Definitely not if the issue is opened with the words "shit library" and the last words are "FUCK YOU". If you ask a question in a respectful manner you will receive a constructive answer.

Well, not worth anyway. If even a simple sample code it's an hair pulling mess which produce black images I don't even want to imagine what the """"solution"""" would be. It was only for your own good so someone woul actually use this library with success.

Headed to freespire.barcode with much happiness and productivity, 3 minutes and I already got QR codes and Barcodes printed out happily, while with this library (trying to not use harsh words here since you're so sensitive and easily offended, though I had quite few better definitions other than "library") i pulled my hair for more than 1 hour with all the examples on google and all I got was a mess

Check on google/stack overflow issue questions with your library and with spire.barcode, you are clever enough to see what the results are.... then come back here denying reality once again to me, I love it!

My advice was for you to improve, even if put straight to your face instead of all the flashy and candy words other people use. No problem for me.

Farewell, my friend,
